src/script.js

```diff
@@ -34,6 +34,7 @@ import {
   PAGE_TYPES,
 } from "@paypal/sdk-constants/src";
 
+import { getSessionState } from "./session"; // eslint-disable-line import/no-cycle
```

Comment:
Lets move this logic to an another module or we can even introduce a new module to avoid the cyclic dependency between script and session.